src/Core/IEducation.cs

```diff
@@ -0,0 +1,33 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+namespace SonarLint.VisualStudio.Core
+{
+    /// <summary>
+    /// Education / help-related services
+    /// </summary>
+    public interface IEducation
+    {
+        /// <summary>
+        /// Displays the help for the specific Sonar rule
+        /// </summary>
+        void ShowRuleHelp(SonarCompositeRuleId ruleId);
```

Comment:
Moved the interface to `Core` so it can be used by IssueViz, which handles the tooltip navigation.